Framework/DataHandling/src/BankPulseTimes.cpp

```diff
@@ -54,7 +53,13 @@ BankPulseTimes::BankPulseTimes(const std::vector<Mantid::Types::Core::DateAndTim
 BankPulseTimes::BankPulseTimes(::NeXus::File &file, const std::vector<int> &periodNumbers)
     : startTime(DEFAULT_START_TIME), periodNumbers(periodNumbers), have_period_info(true),
       m_sorting_info(PulseSorting::UNKNOWN) {
-  file.openData("event_time_zero");
+
+  // Some old data use "pulse_time" instead of "event_time_zero" as entry
+  try {
+    file.openData("event_time_zero");
+  } catch (std::exception &e) {
```

Comment:
```suggestion
  } catch (const std::exception &) {
```
should fix that pesky windows compiler warning